k8s/sample-app/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,19 @@
 actions:
-- name: dummy test
+- name: Sample test check port 80
   bashTest:
     script: |-
-      exit 0
+      # Here is a sample test which checks if the Nginx is listening on port 80
+      curl http://$APP_INSTANCE_NAME-sample-app-svc.$NAMESPACE.svc.cluster.local:80
     expect:
-      exitCode:
-        equals: 0
+      stdout:
+        contains: 'Welcome to nginx'
+- name: Site address should be 200 OK
+  httpTest:
+    url: http://$APP_INSTANCE_NAME-sample-app-svc.$NAMESPACE.svc.cluster.local
```

Comment:
I believe this should be something like `url: http://{{ .Env.APP_INSTANCE_NAME }}-sample-app-svc.{{ .Env.NAMESPACE }}.svc.cluster.local`.

For more examples please check [PR#382](https://github.com/GoogleCloudPlatform/click-to-deploy/pull/382/files).